docs/commands/build-docs.md

```diff
@@ -0,0 +1,73 @@
+# `build-docs`
+
+## Introduction
+
+The `build-docs` command builds Redoc into a zero-dependency HTML file.
+
+## Usage
+
+```bash
+redocly build-docs <api>
+redocly build-docs <api> --output=custom.html
+redocly build-docs <api> --features.openapi.theme.colors.primary.main=orange
+redocly build-docs <api> -t custom.hbs
+redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page meta description"
+```
+
+
+## Options
+
+Option | Type | Description
+-- | -- | --
+api | string | Path to the API definition filename or alias that you want to generate the preview for. Refer to [the api section](#api) for more options.
```

Comment:
Is this required?

Reply:
What is "the api section" and there is none that I found.

Reply:
> Is this required?

No. We support running `build-docs` command in the project directory if this directory contains a config file. We try to find the config file in the working directory.
```
redocly build-docs
```

> What is "the api section" and there is none that I found.

Added API section to documentation, like we have in `preview-docs` command.